errors/daemon.go

```diff
@@ -715,6 +715,15 @@ var (
 		HTTPStatusCode: http.StatusConflict,
 	})
 
+	// ErrorCodeContainerRestarting is generated when we try to top the
+	// container, but the container is restarting.
+	ErrorCodeContainerRestarting = errcode.Register(errGroup, errcode.ErrorDescriptor{
+		Value:          "CONTAINERRESTARTING",
+		Message:        "Container %s is restarting, wait until the container is running",
+		Description:    "An attempt was made to top the container, but the container is restarting",
```

Comment:
I agree, we should be more generic that "an operation was made on a restarting container" (or something like that).